bcda/responseutils/writer.go

```diff
@@ -22,4 +23,99 @@ func WriteError(outcome *fhirmodels.OperationOutcome, w http.ResponseWriter, cod
 	}
 }
 
-func WriteCapabilityStatement() {}
+func CreateCapabilityStatement() *fhirmodels.CapabilityStatement {
+	usecoors := true
+	dt := time.Now()
+	baseurl := "<host>"
+	statement := &fhirmodels.CapabilityStatement{
+		Status:       "active",
+		Date:         &fhirmodels.FHIRDateTime{Time: dt, Precision: fhirmodels.Date},
+		Publisher:    "Centers for Medicare & Medicaid Services",
+		Kind:         "capability",
+		Instantiates: []string{"https://api.bluebutton.cms.gov/v1/fhir/metadata"},
+		Software: &fhirmodels.CapabilityStatementSoftwareComponent{
+			Name:        "Beneficiary Claims Data API",
+			Version:     "0.1",
```

Comment:
Should version number be configurable for dynamic updating?

Reply:
I like it! I will change this as well